GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -716,9 +729,23 @@ private static String dateCellValue(String value) {
         return ExportImportUtils.formatDateResponse(value);
     }
 
-    private static String photoCellValue(String value, String imagePrefix) {
-        return ExportImportUtils.formatImage(imagePrefix, value);
-    }
+    private static List<String> mediaCellValues(String value, boolean useQuestionId, String imagePrefix) {
+        List<String> cells = new ArrayList<>();
+        if (value.startsWith("{")) { //JSON, possibly geotagged
```

Comment:
The previous code assumed that photo answers were always in JSON, so yes, the if clause is probably redundant.